android/engine/src/main/java/org/smartregister/fhircore/engine/sync/SyncListenerManager.kt

```diff
@@ -111,7 +120,8 @@ constructor(
           ConfigurationRegistry.ORGANIZATION -> authenticatedUserInfo?.organization
           ConfigurationRegistry.PUBLISHER -> authenticatedUserInfo?.questionnairePublisher
           ConfigurationRegistry.ID -> paramExpression
-          ConfigurationRegistry.COUNT -> appConfig.count
+          //                    ConfigurationRegistry.COUNT -> appConfig.count
```

Comment:
Why are we not fetching the count from the configs anymore?

Reply:
@sevenreup that was not supposed to be case